bucket/ontopreplica.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/LorenzCK/OnTopReplica",
+    "description": "A real-time always-on-top “replica” of a window",
+    "version": "3.5.1-fix",
+    "license": "MIT",
+    "url": "https://github.com/LorenzCK/OnTopReplica/releases/download/v3.5.1-fix/OnTopReplica-v3.5.1-fix.zip",
+    "hash": "3206460ce10eb563ac3acc600cb2d7143a569b745fd2a11937aac2d60dba5236",
+    "installer": {
+        "script": [
+            "Remove-Item \"$dir\\setup.exe\", \"$dir\\postinstaller.exe\", \"$dir\\postinstaller.exe.config\"",
+            "Expand-MsiArchive \"$dir\\OnTopReplica.msi\" -Removal"
+        ]
+    },
+    "shortcuts": [
+        [
+            "OnTopReplica.exe",
+            "OnTopReplica"
+        ]
+    ],
+    "persist": "OnTopReplica.exe.config"
```

Comment:
I don't think this is the app's config file. This might be related to the binary itself.

According to @wenmin92's comment, the config file is present in `%LocalAppData%\Lorenz_Cuno_Klopfenstein`. See also https://github.com/LorenzCK/OnTopReplica/issues/83#issuecomment-488308479

Reply:
You are right. After tweaking some config, I found that nothing is changed in `OnTopReplica.exe.config.`.